self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryProjectManagersTestCase.java

```diff
@@ -25,8 +22,9 @@ public void getProjectManagerById() {
         final Storage storage = new InMemory();
         InMemoryProjectManagers inMemoryPm =
                 new InMemoryProjectManagers(storage);
-        assertThat(inMemoryPm.getById(1),
-                is(inMemoryPm.pick("github")));
+
+        assertThat(inMemoryPm.getById(1).id(), is(
+                (storage.projectManagers().getById(1)).id()));
```

Comment:
Nu sunt sigur ca e ok asta, dar altfel nu stiu cum as putea verifica, avand in vedere ca inMemoryPm.getById(1) si storage.projectManagers().getById(1) au hashcode-uri diferite.

Reply:
@HoreaPetrila ``ProjectManaged.id()`` returneaza Integer, poti verifica simplu cu ``equalTo``.

Reply:
@amihaiemil Done